### PR TITLE
confirm deposit route fixes AB#789

### PR DESCRIPTION
**Pull request by josipkoncurat**

### CODE REVIEW

File: src/docs/asciidoc/index.adoc [hunk 1]
```diff
@@ -7,6 +7,7 @@ Version {version}, {date}
 :source-highlighter: prettify
 :docinfo: shared
 :icons: font
+:icons: font
```
Comment (githinho) [inline src/docs/asciidoc/index.adoc:10]: Why double?

File: src/main/kotlin/com/ampnet/walletservice/service/impl/DepositServiceImpl.kt [hunk 2]
```diff
@@ -94,6 +94,15 @@ class DepositServiceImpl(
         return depositRepository.findAllByOwnerUuid(user).map { DepositServiceResponse(it, true) }
     }
 
+    @Transactional
+    override fun confirm(id: Int): DepositServiceResponse {
+        val deposit = depositRepository.findById(id).orElseThrow {
+            ResourceNotFoundException(ErrorCode.WALLET_DEPOSIT_MISSING, "Missing deposit with id: $id")
+        }
+        deposit.userConfirmation = true
+        return DepositServiceResponse(deposit)
```
Comment (githinho) [inline src/main/kotlin/com/ampnet/walletservice/service/impl/DepositServiceImpl.kt:103]: Verify that this deposit belongs to the user.

**Review by githinho — CHANGES_REQUESTED** [review]
Also, add confirm for project deposit.

**Comment by githinho** [comment]
> Also, add confirm for project deposit.

👆 This is still missing.

**Comment by josipkoncurat** [comment]
> > Also, add confirm for project deposit.
> 
> 👆 This is still missing.

Oh, I missed it, will do it.

File: src/test/kotlin/com/ampnet/walletservice/controller/ControllerTestBase.kt [hunk 3]
```diff
@@ -207,12 +207,13 @@ abstract class ControllerTestBase : TestBase() {
...
         txHash: String = "th_ktDw9ytaQ9aSi78qgCAw2JhdzS8F7vGgzYvWeMdRtP6hJnQqG",
-        coop: String = COOP
+        coop: String = COOP,
+        createdBy: UUID = userUuid
     ): Deposit {
         val document = saveFile("doc", "document-link", "type", 1, owner)
         val deposit = Deposit(
             0, owner, "S34SDGFT", amount,
-            ZonedDateTime.now(), userUuid, type, txHash, userUuid,
+            ZonedDateTime.now(), userUuid, type, txHash, createdBy,
```
Comment (githinho) [inline src/test/kotlin/com/ampnet/walletservice/controller/ControllerTestBase.kt:216]: This looks wrong to me. First `userUuid` was for value createdBy and the second was for value approvedBy.
```suggestion
            ZonedDateTime.now(), createdBy, type, txHash, userUuid,
```

If the tests are passing and one test to verify unauthorized confirm deposit.

**Review by githinho — CHANGES_REQUESTED** [review]

**Review by githinho — APPROVED** [review]

**Pull request merged**